README.md

```diff
@@ -1,46 +1,138 @@
       <img src="https://img.shields.io/badge/MADE%20FOR%20Apostrophe%203-000000.svg?style=for-the-badge&logo=Apostrophe&labelColor=6516dd">
     </a>
-    <a aria-label="Test status" href="https://github.com/apostrophecms/apostrophe/actions">
-      <img alt="GitHub Workflow Status (branch)" src="https://img.shields.io/github/workflow/status/apostrophecms/apostrophe/Tests/main?label=Tests&labelColor=000000&style=for-the-badge">
+    <a aria-label="Test status" href="https://github.com/apostrophecms/cache-redis/actions">
+      <img alt="GitHub Workflow Status (branch)" src="https://img.shields.io/github/workflow/status/apostrophecms/cache-redis/Tests/main?label=Tests&labelColor=000000&style=for-the-badge">
     </a>
     <a aria-label="Join the community on Discord" href="http://chat.apostrophecms.org">
       <img alt="" src="https://img.shields.io/discord/517772094482677790?color=5865f2&label=Join%20the%20Discord&logo=discord&logoColor=fff&labelColor=000&style=for-the-badge&logoWidth=20">
     </a>
-    <a aria-label="License" href="https://github.com/apostrophecms/module-template/blob/main/LICENSE.md">
+    <a aria-label="License" href="https://github.com/apostrophecms/cache-redis/blob/main/LICENSE.md">
       <img alt="" src="https://img.shields.io/static/v1?style=for-the-badge&labelColor=000000&label=License&message=MIT&color=3DA639">
     </a>
   </p>
 </div>
 
-This module template serves as a starting point for new official Apostrophe modules. This is where you would describe what the purpose of the module is.
+This module enhances the core caching module, `@apostrophecms/cache`, to use [Redis](https://redis.io/) rather than MongoDB. This module does not set up the actual Redis store, but instead allows Apostrophe to access an existing Redis store through the standard Apostrophe caching API and an internal Redis client.
+
+All normal Apostrophe cache API features are maintained in addition to the Redis-specific features described below.
 
 ## Installation
 
 To install the module, use the command line to run this command in an Apostrophe project's root directory:
 
 ```
-npm install @apostrophecms/module-template
+npm install @apostrophecms/cache-redis
 ```
 
 ## Usage
 
-Configure the _______ module in the `app.js` file:
+Configure the Redis cache module in the `app.js` file:
 
 ```javascript
 require('apostrophe')({
   shortName: 'my-project',
   modules: {
-    '@apostrophecms/module-template': {}
+    '@apostrophecms/cache-redis': {}
   }
 });
 ```
 
-### Additional usage sections
+## Configuring Redis options
+
+**All options for this module should be applied to `@apostrophecms/cache` in project code.** This module simply "improves" that core module (updates its features). The main caching module still does all the work.
+
+Configure the underlying Redis client by adding a configuration object on the cache module's `redis` option.
+
+```javascript
+// modules/@apostrophecms/cache-redis/index.js
```

Comment:
This path is wrong, right? Per your note above about configuring `@apostrophecms/cache` rather than the improvement.

Reply:
ah yes. you're right.